airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/upgrade_cdk/pipeline.py

```diff
@@ -30,14 +33,22 @@ def __init__(
 
     async def _run(self) -> StepResult:
         context = self.context
-        og_connector_dir = await context.get_connector_dir()
-        if "setup.py" not in await og_connector_dir.entries():
-            return self.skip("Connector does not have a setup.py file.")
-        setup_py = og_connector_dir.file("setup.py")
-        setup_py_content = await setup_py.contents()
+
         try:
-            updated_setup_py = self.update_cdk_version(setup_py_content)
-            updated_connector_dir = og_connector_dir.with_new_file("setup.py", updated_setup_py)
+            og_connector_dir = await context.get_connector_dir()
+            if self.context.connector.language == ConnectorLanguage.PYTHON or self.context.connector.language == ConnectorLanguage.LOW_CODE:
+                updated_connector_dir = await self.upgrade_cdk_version_for_python_connector(og_connector_dir)
+            elif self.context.connector.language == ConnectorLanguage.JAVA:
```

Comment:
To check if something matches an enum value please use `is` instead of `==`. 
It's equivalent but I think its recommended by our linter and I find it more pythonic.
`elif self.context.connector.language is ConnectorLanguage.JAVA`